services/fluxmonitor.go

```diff
@@ -130,22 +130,16 @@ func NewFluxMonitor(job string, config FluxMonitorConfig, triggerJobRun chan sub
 
 	FAEvents := make(chan interface{})
 
-	state, err := fm.blockchain.Request(common.FMRequestState)
+	state, err := fm.blockchain.GetState(context.TODO())
```

Comment:
How do you envision this `context.TODO()` to be used?

Reply:
We'll define the request timeouts here, instead of having the blockchains define them. I'll make an issue for this.

Reply:
#141